docs/explanation/which-queries-to-use.md

```diff
@@ -0,0 +1,43 @@
+# Which WPGraphQL endpoints to use: /graphql vs ?graphql
+
+WPGraphQL exposes a GraphQL endpoint that developers can use to interact with the WordPress backend and retrieve data.
+
+The plugin provides two primary ways to interact with this API:
+
+1. The pretty URL endpoint `/graphql`
+2. The query-string-based endpoint `/index.php?graphql`
+
+Both serve the same purpose of providing access to the GraphQL API, but they function slightly differently.
+
+## The default `/graphql` endpoint
+
+The `/graphql` default endpoint is the preferred choice for most users because it is easy to read, share, and remember.
+
+WPGraphQL allows you to customize the `/graphql` endpoint using a filter. If you'd like to change the endpoint to something more specific to your site, you can do so with a simple code snippet.
+
+```php
+function my_new_graphql_endpoint() {
+  return 'my_endpoint';
+};
+
+add_filter( 'graphql_endpoint', 'my_new_graphql_endpoint' );
+```
+This code would change the default `/graphql` endpoint to `/my_endpoint`.
```

Comment:
Maybe we put this in its own section? And should we also mention this can be configured through the WPGraphQL settings?